low-level/framework/utils/utility.py

```diff
@@ -156,6 +158,56 @@ def get_current_time():
         """Returns the time as integer number in seconds since the epoch in UTC."""
         return int(time.time())
 
+    @staticmethod
+    def get_manufacturer_name():
+        """Returns node server manufacturer name."""
+        manufacturer = "NA"
+        cmd = "ipmitool bmc info"
+        res_op, _, res_rc = SimpleProcess(cmd).run()
+        if isinstance(res_op, bytes):
+            res_op = res_op.decode("utf-8")
+        if res_rc == 0:
+            search_res = re.search(
+                r"Manufacturer Name[\s]+:[\s]+([\w]+)(.*)", res_op)
+            if search_res:
+                manufacturer = search_res.groups()[0]
+        return manufacturer
+
+    @staticmethod
+    def get_server_details():
+        """Returns a dictionary of server information.
+
+        Grep 'FRU device description on ID 0' information using ipmitool command.
+        """
+        specifics = {
+            "Board Mfg": "NA",
+            "Board Product": "NA",
+            "Board Part Number": "NA",
+            "Product Name": "NA",
+            "Product Part Number": "NA",
+            "Manufacturer": Utility.get_manufacturer_name(),
+            "OS": Utility.get_os()
+            }
+        cmd = "ipmitool fru print"
```

Comment:
same comment as like line no 165

Reply:
@satish-seagate. I addressed it.
Please review.